engine/collection/synchronization/engine.go

```diff
@@ -447,14 +401,16 @@ func (e *Engine) sendRequests(ranges []flow.Range, batches []flow.Batch) error {
 			Nonce:    rand.Uint64(),
 			BlockIDs: batch.BlockIDs,
 		}
-		err := e.conduit.Multicast(req, synccore.DefaultBlockRequestNodes, e.participants.NodeIDs()...)
+		err := e.con.Multicast(req, synccore.DefaultBlockRequestNodes, e.participants.NodeIDs()...)
 		if err != nil {
-			errs = multierror.Append(errs, fmt.Errorf("could not submit batch request (size=%d): %w", len(batch.BlockIDs), err))
```

Comment:
Maybe we could add the more informational error message back (and add to consensus engine as well)